CARTA-AppImage-creator/Dockerfile-carta-appimage-create-Almalinux8

```diff
@@ -0,0 +1,182 @@
+# Dockerfile to build and package a universal Linux Appimage of CARTA.
+# Built on our oldest supported platform (Almalinux8.10/RHEL7) to ensure full forward compatibility.
+# Requires BASE_IMAGE, ARCH_TYPE, CASACORE_TAG, BACKEND_TAG, FRONTEND_TAG, and VERSION arguments from the 'create-carta-appimage.sh' script.
+
+ARG BASE_IMAGE
+# On x86_64 it uses "almalinux:8.10", on aarch64 it uses "arm64v8/almalinux:8.10"
+FROM $BASE_IMAGE
+
+ARG ARCH_TYPE
+ARG CASACORE
+ARG NPM
+ARG FRONTEND
+ARG BACKEND
+ARG RELEASE_TAG
+ARG APPIMAGE_VER
+
+# FROM almalinux:8.10
+# Install the required packages
+RUN \
+  dnf -y install epel-release && \
+  dnf -y install 'dnf-command(config-manager)' && \
+  dnf -y config-manager --set-enabled powertools && \
+  dnf -y update && \
+  dnf -y install autoconf automake bison blas-devel bzip2 cmake curl-devel flex fftw fftw-devel gcc \
+         gcc-c++ gcc-gfortran git git-lfs gsl-devel hdf5-devel lapack-devel libasan \
+         libtool libxml2-devel libzstd-devel libuuid-devel make ncurses-devel openssl-devel patchelf protobuf-devel \
+         pugixml-devel python36 python3-pip readline-devel subversion \
+         unzip wcslib-devel wget zlib-devel libuuid-devel zfp-devel && \
+  pip3 install numpy astropy
```

Comment:
It seems this command line failed in my docker image build. The executor `pip3` is from Python version 3.6, and now it can not install `numpy` and `astropy` modules. If I replace it with a newer version, e.g. Python 3.9, the problem can be solved. I am also wondering do we need to install these two Python packages. It seems they are only used in the `fits2idia` test script. If we want to use these Python packages, we may also need to copy additional Python binary files when packaging (?) But I think this is not a critical issue. We can test to remove this command line and see if it has any side effects.

Reply:
I think that changing to the newer python version should be fine. Because the carta-casacore currently is `-DBUILD_PYTHON=OFF`.
Currently we don't using any Python package in our carta-backend, so we do not include any python lib. 
Please test python3.9, and generating the AppImage (KS can test them), if there is no regression problem, I think update the Python version should be reasonable.